build/Dockerfile

```diff
@@ -23,3 +23,5 @@ COPY job-templates/* /default-job-templates/
 COPY _output/bin/eunomia ${OPERATOR}
 
 ENTRYPOINT ["/usr/local/bin/eunomia-operator"]
+
+USER 12345
```

Comment:
Does this work without the UID even existing?
On openshift it gets overwritten by the platform, but it might now work an a plain K8s (like GKE)

Reply:
Yes it does. As per docker documentation it is possible to just provide UID: https://docs.docker.com/engine/reference/builder/#user. Also I tested it locally on minikube and I haven't encountered any problems.